README.md

```diff
@@ -52,131 +40,37 @@ For an example, see `application.yml.example`
-
-### MVI Service
-The Master Veteran Index Service retreives and updates a veterans 'golden record'.
-Update the `MVI_URL` env var in config/application.yml with the value given to you
-by devops or your team.
-```
-# config/application.yml
-MVI_URL = '...'
-```
-Since that URL is only accessible over the VA VPN a mock service is included in the project.
-To enable it set MOCK_MVI_SERVICE in config/application.yml to 'true'
-```
-# config/application.yml
-MOCK_MVI_SERVICE = true
-```
-Endpoint response values can be set by copying mock_mvi_responses.yml.example to
-mock_mvi_responses.yml. For the find_candidate
-endpoint you can return different responses based on SSN:
-```
-find_candidate:
-  555443333:
-    birth_date: '19800101'
-    edipi: '1234^NI^200DOD^USDOD^A'
-    family_name: 'Smith'
-    gender: 'M'
-    given_names: ['John', 'William']
-    icn: '1000123456V123456^NI^200M^USVHA^P'
-    mhv_id: '123456^PI^200MHV^USVHA^A'
-    ssn: '555443333'
-    status: 'active'
-  111223333:
-    # another mock response hash here...
-```
+1. `postgres -D /usr/local/var/postgres`
+1. `redis-server /usr/local/etc/redis.conf`
+1. `bundle exec rails` server from <GITHUB_HOME>/vets-api/
 
-### Running the App
+#### Running the App with Foreman
 1. Start the application: `foreman start`
 1. Navigate to <http://localhost:3000/v0/status> in your browser.
 
-## Testing Commands
+### Testing Commands
 - `bundle exec rake lint` - Run the full suite of linters on the codebase.
 - `bundle exec guard` - Runs the guard test server that reruns your tests after files are saved. Useful for TDD!
 - `bundle exec rake security` - Run the suite of security scanners on the codebase.
 - `bundle exec rake ci` - Run all build steps performed in Travis CI.
 
-### Manually Testing ID.me Authentication Flow
-The first endpoint, below, doesn't require authentication while the second does:
-```
-curl localhost:3000/v0/status
-curl localhost:3000/v0/welcome
-```
-
-The callback from ID.me is configured to go to `http://localhost:3001/auth/login/callback`, which is a front-end route in production. To test just the API locally, without running the vets-website server, start the vets-api server on port 3001:
-```
-bundle exec rails s -p 3001
-```
-Curl or browse to `http://localhost:3001/v0/sessions/new`; copy and paste the ID.me URL into your browser. Create your ID.me account if you have not already done so (**Note**: creating your account on the ID.me site is separate from the api.idmelabs.com sandbox) or sign in with your username and password.
-
-The token returned in the json response at the end of the login flow can be used as follows (You may wish to use Postman instead of curl to test within the browser):
-
-```
-curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyyja" localhost:3001/v0/sessions/current
-curl --header "Authorization: Token token=GvmkAW231VxGHkYxyppr2QQsi1D7PStqeiJXyyja" localhost:3001/v0/profile
-```
-
 ## Deployment Instructions
```

Comment:
I think it's worth a pointer to the devops repo (https://github.com/department-of-veterans-affairs/devops#vets-api), but I like the instructions for determining the currently deployed version (frequently the first step in troubleshooting an issue). Details on the deployment pipeline are probably not much use here.